File: JavaToturial/data_structure/src/array_DSA/selection_sort.java
package array_DSA;

public class selection_sort {
public static void main(String[] args) {
	int a[]= {2,9,3,5,10,0};
	int n=6;
int min, i,j,swap;

for(i=0; i<n; i++) {
	min=i;
	for(j=i+1; j<n; j++) {
		if(a[min]>a[j]) {
			min=j;
		}
	}
		if(i!=min) {
			swap=a[i];
			a[i]=a[min];
			a[min]=swap;
		}	
}
System.out.println("your sorted array:");
for(int e:a) {
	System.out.println(e);
}
}
}

File: collage_project/c practic/hemant sir program/selection sort.c
#include<stdio.h>
#include<conio.h>
void main()
{
	int a[5]={21, 9,5,111,-98}, i, j,key;
		for(i=0; i<5; i++)
	{
		printf("your unsorted array= %d\n",a[i]);
	}
	

	
	for(i=0; i<5; i++)
	{
		key=i;
		for(j=i+1; j<5; j++)
		{
			if(a[i]>a[j])
			{
				key=j;
			}
		}
		if(key !=i)
		{
			int swap;
			swap= a[key];
			a[key]=a[i];
			a[i]=swap;
		}
	}
	
	
	
	
	for(i=0; i<5; i++)
	{
		printf("\n");
	}
	for(i=0; i<5; i++)
	{
		printf("your sorted array= %d\n",a[i]);
	}
	getch();
}